src/lib/routes/admin-api/tag.test.ts

```diff
@@ -72,15 +72,15 @@ test('should get all tags added', () => {
         });
 });
 
-test('should be able to get single tag by type and value', () => {
+test('should be able to get single tag by type and value', async () => {
     expect.assertions(1);
-    tagStore.createTag({ value: 'TeamRed', type: 'simple' });
+    await tagStore.createTag({ value: 'TeamRed', type: 'simple' });
     return request
         .get(`${base}/api/admin/tags/simple/TeamRed`)
         .expect('Content-Type', /json/)
         .expect(200)
         .expect((res) => {
-            expect(res.body.tag.value).toBe('TeamRed');
```

Comment:
Was the test wrong or did we change the API?

Reply:
I guess it meant that we changed the API, I'll revert and create a schema for this response, so we don't need a major version for this PR